examples/environment-guide/README.md

```diff
@@ -0,0 +1,88 @@
+Environment Guide Example
+=========================
+This example demonstrates running pods, replication controllers, and
+services. It shows two types of pods: frontend and backend, with
+services on top of both. Accessing the frontend pod will return
+environment information about itself, and a backend pod that it has
+accessed through the service. The goal is to illuminate the
+environment metadata available to running containers inside the
+Kubernetes cluster. The documentation for the kubernetes environment
+is [here](/docs/container-environment.md).
+
+![Diagram](diagram.png)
+
+Prerequisites
+-------------
+This example assumes that you have a Kubernetes cluster installed and
+running, and that you have installed the `kubectl` command line tool
+somewhere in your path.  Please see the [getting
+started](/docs/getting-started-guides) for installation instructions

```

Comment:
Does that link work? In general, we've been using relative links: ../../docs/getting-started-guides